packages/ods/src/components/timepicker/src/controller/ods-timepicker.ts

```diff
@@ -0,0 +1,53 @@
+import { ODS_TIMEZONES_PRESET } from '../constant/timezone-preset';
+import { type ODS_TIMEZONE, ODS_TIMEZONES, type OdsTimezone } from '../constant/timezones';
+
+function formatValue(value?: string, withSeconds?: boolean): string {
+  if (!withSeconds && value?.match(/:/g)?.length === 2) {
+    const inputValue = value.split(':');
+    return `${ inputValue[0] }:${ inputValue[1] }`;
+  }
+
+  if (withSeconds && value?.match(/:/g)?.length === 1) {
+    return `${ value }:00`;
+  }
+
+  return '';
+}
+
+function getCurrentTimezone(currentTimezone?: OdsTimezone): OdsTimezone | undefined {
+  if (!currentTimezone) {
+    const browserTimezone = new Date().getTimezoneOffset() / 60 * -1;
+    const parsedTimezone = browserTimezone >= 0 ? `+${browserTimezone}` : browserTimezone.toString();
+    return ODS_TIMEZONES.find((timezone) => timezone.indexOf(parsedTimezone) > -1);
+  }
+  return currentTimezone;
+}
+
+function parseTimezones(timezones?: OdsTimezone[] | ODS_TIMEZONES_PRESET | string): ODS_TIMEZONE[] {
```

Comment:
probably easier to return `OdsTimezone[]` instead and manipulate the same type everywhere